sickbeard/scene_exceptions.py

```diff
@@ -320,7 +320,7 @@ def _xem_exceptions_fetcher():
             if not parsedJSON['data']: 
                 logger.log(u"No data returned from XEM when checking scene exceptions. Update failed for {0}".format
                             (sickbeard.indexerApi(indexer).name), logger.DEBUG)
-                continue
+                break
```

Comment:
Also: {"result":"success","data":[],"message":""}.. looks too me dat it did not get a result for that search. I would at least try the others. This does not look like an error that you get when the service is down? @labrys your thoughts?